js/screens/Onboarding/Onboarding.js

```diff
@@ -50,73 +44,72 @@ let deviceWidth = Dimensions.get('window').width;
 
 const OnboardingPage = () => {
   return (
-    <View>
-      <Onboarding
-        bottomBarHighlight={false}
-        imageContainerStyles={{
-          width: deviceWidth,
-          paddingBottom: 40,
-          paddingTop: 70,
-          justifyContent: 'flex-start',
-        }}
-        bottomBarHeight={100}
-        flatlistProps={{
-          marginHorizontal: 60,
-          width: deviceWidth,
-        }}
-        DotComponent={Circle}
-        SkipButtonComponent={Skip}
-        pages={[
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/tutorial.png')}
-              />
-            ),
-            title: <Title title="View updates on your child" />,
-            subtitle: <Subtitle subtitle="Description to be added" />,
-            bottomBarHeight: 100,
-          },
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/fox.png')}
-              />
-            ),
-            title: <Title title="Set up a personalized profile" />,
-            subtitle: <Subtitle subtitle="Description to be added" />,
-          },
-          {
-            backgroundColor: '#76D2D1',
-            image: (
-              <Image
-                style={{ height: 219, width: 219 }}
-                source={require('../../assets/login/provider.png')}
-              />
-            ),
+    <Onboarding
```

Comment:
Please move all your inline styling from here and put it in the styles.js file. Thank you.